File: db4o.net/drs/Db4objects.Drs.Tests/Db4objects.Drs.Tests/CollectionAssert.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using System.Collections;
using Db4oUnit;
using Db4objects.Drs.Tests;

namespace Db4objects.Drs.Tests
{
	public class CollectionAssert
	{
		public static void AreEqual(IEnumerable expected, IEnumerable actual)
		{
			Iterator4Assert.AreEqual(CollectionAssert.Adapt(expected.GetEnumerator()), CollectionAssert
				.Adapt(actual.GetEnumerator()));
		}

		internal static IEnumerator Adapt(IEnumerator iterator)
		{
			return ReplicationTestPlatform.Adapt(iterator);
		}
	}
}

File: drs/src/test/com/db4o/drs/test/CollectionAssert.java
package com.db4o.drs.test;

import java.util.*;

import com.db4o.foundation.*;

import db4ounit.*;

public class CollectionAssert {

	public static void areEqual(Iterable expected, Iterable actual) {
		Iterator4Assert.areEqual(CollectionAssert.adapt(expected.iterator()), CollectionAssert.adapt(actual.iterator()));
	}

	static Iterator4 adapt(Iterator iterator) {
		return ReplicationTestPlatform.adapt(iterator);
	}

}
